test-dev/__snapshots__/dev.test.ts.snap

```diff
@@ -47,7 +49,7 @@ exports[`snowpack dev smoke: js 1`] = `
  * When you're ready to start on your site, clear the file. Happy hacking!
  **/
 
-import confetti from '../_snowpack/pkg/canvas-confetti.js';
+import confetti from '../_snowpack/pkg/canvas-confetti.v1.3.2.js';
```

Comment:
Little bikeshed: would `@v1.3.2` be a little easier to read / parse out instead of a sub-extension?